Makefile

```diff
@@ -2,10 +2,13 @@ ZPOOL=""
 SERVER=""
 SVN_REL_URL=https://svn.freebsd.org/base/releng/11.1
```

Comment:
The version release version can be obtained dynamically 

```
# uname -r | cut -d "-" -f 1
11.1
```

Reply:
This is still dangerous, they not be using a RELEASE, but rather anything like BETA, STABLE or even CURRENT.

Reply:
What's the best way to figure out the remote SVN branch?

Reply:
I think it makes sense that CURRENT guys get `head`, STABLE/BETA/RC/* get `stable` and RELEASE is `releng`.

Reply:
hence my supplying only a variable you can override, instead of trying to be smart.

Reply:
Ok, we can revisit this in #95

Reply:
```shell
case $(cut -d "-" -f 2) in
	RELEASE)
		branch="releng"
		;;
	CURRENT)
		branch="head"
		;;
	*)
		branch="stable"
		;;
esac
```

Or something